src/views/home-postpage.js

```diff
@@ -0,0 +1,50 @@
+import { postPage } from './postpageProfile.js';
+import { goNewPost } from './modal-newpost.js';
+
+export function goHome() {
+  firebase.firestore().collection('posts').onSnapshot((snap) => {
+    root.innerHTML = ' ';
+    let post = ' ';
+    post = document.createElement('div');
+    root.innerHTML = `<div id="navBar">
+    <img class="is-rounded"  src="https://uxrecipe.github.io/img/uxrecipe-logo.png" id="logo">
+    <div class="dropdown">
+    <button class="dropbtn"><i class="fas fa-bars" aria-hidden="true"></i></button>
+    <div class="dropdown-content">
+    <a href="#"><i class="fas fa-bell" aria-hidden="true"></i> Notificaciones</a>
+    <a href="#"><i class="fas fa-bookmark" aria-hidden="true"></i> Guardado</a>
+    <a href="#"><i class="fas fa-wrench" aria-hidden="true"></i> Configuracion</a>
+    </div>
+    </div>
+    </div>
+    <br>
```

Comment:
Cada vez que se usa br muere un gatito :´´(   no usen br mejor usen salto de párrafo
```suggestion
    <p/>
```